src/shared/components/TextArea/TextArea.style.ts

```diff
@@ -11,6 +11,7 @@ export const StyledTextArea = styled.textarea`
   resize: none;
   color: ${colors.gray[300]};
   font-size: ${typography.sizes.subtitle2};
+  line-height: 2;
```

Comment:
Per design text here is `Body 1`, so we should use values from the theme that correspond to it
```suggestion
  font-size: ${typography.sizes.body1};
  line-height: ${typography.lineHeights.body1};
```